packages/core/src/notification/selector/types/cdevents/CDEventsNotificationType.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import type { INotificationTypeCustomConfig } from '../../../../domain';
+import { FormikFormField, TextInput, Validators } from '../../../../presentation';
+
+export class CDEventsNotificationType extends React.Component<INotificationTypeCustomConfig> {
+  public render() {
+    const { fieldName } = this.props;
+    return (
+      <>
+        <FormikFormField
+          label="Events Broker URL"
+          name={fieldName ? `${fieldName}.address` : 'address'}
+          validate={Validators.skipIfSpel(Validators.urlValue('Please enter a valid URL'))}
+          input={(props) => (
+            <TextInput
+              inputClassName={'form-control input-sm'}
+              {...props}
+              placeholder="URL starts with https://events-broker-address/default/events-broker/"
+            />
+          )}
+          required={true}
+        />
+        <FormikFormField
+          label="CDEvents Type"
+          name={fieldName ? `${fieldName}.cdEventsType` : 'cdEventsType'}
+          validate={Validators.skipIfSpel(Validators.cdeventsTypeValue('Please enter a valid CDEvents Type'))}
+          input={(props) => (
+            <TextInput
+              inputClassName={'form-control input-sm'}
+              {...props}
+              placeholder="CDEvents Type starts with dev.cdevents.<subject>.<predicate>"
```

Comment:
when looking at other notifications I think placeholders are showing an example/description how actual value should be, in this case the input for `CDEvents Type` can have different type of events from [CDEvents spec](https://github.com/cdevents/spec/tree/v0.3.0) and starts with `dev.cdevents.`

Reply:
The CDEvents spec details that event types "should" start with `dev.cdevents`, not "must".  Since the spec allows this to be configurable, we should also allow that to be configurable.

That's the typical interpretation of that language in other RFC processes.

As for the placeholder value itself, I don't have a strong opinion on the text of it, so long as it doesn't conflict with event type validation.

Reply:
Agree, changing the placeholder for CDEvents Type to a message "Enter a CDEvents type" and make configurable through `settings`